CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## 5.3.4-SNAPSHOT (future release)
+## 5.4.0-SNAPSHOT (future release)
 
 **Highlights**
+- Introduce BulkWorkflow which can be used or extended to handle mass of child workflows without overloading the system
+- Allow child workflows to automatically wake up parent workflow instance when they enter end state (can be enabled via workflow settings)
```

Comment:
no longer "can be enable via workflow settings" but instead "when parent is in _wait_ state"